src/app/shared/models/constants/transalation.json.ts

```diff
@@ -1,1847 +1,2008 @@
 export const HindiLang = [
   {
-    "text": "(Effective 2nd September 2018. Last Revised 13th August 2019)",
-    "translation": "(2 सितम्बर 2018 से लागु. अंतिम संशोधित तिथि 13 अगस्त 2019)"
+    text: '(Effective 2nd September 2018. Last Revised 13th August 2019)',
```

Comment:
@Harshwardhan18 Can you disable the formatting here? This is a json data, the keys and values ideally need to be in double quotes